src/styles/navigation-bar.css

```diff
@@ -18,3 +18,14 @@
   font-weight: bold;
   font-size: 16px;
 }
+
+.navbar-link-right {
+  float: right;
+  margin-right: 1.5%;
+  color: black;
+}
+
+.navbar-link-right:hover {
+  text-decoration: none;
+  color: rgba(0, 0, 0, 0.61);
```

Comment:
I think we should change this to opacity. So we only need to rely on the base color from `navbar-link-right`, then simply change the opacity on hover to make both the text and image appear to be lighter... Ex:

```css
.navbar-link-right {
  float: right;
  margin-right: 1.5%;
  color: black;
}
.navbar-link-right:hover {
  text-decoration: none;
  opacity: 0.7;
}
```